<launch>
 <!-- Navigation Program   -->
    <arg name="distance_laser" default="0.3" />
    <arg name="speed_factor" default="0.1" />
    <arg name="forward_speed" default="2" />
    <arg name="backward_speed" default="-2" />
    <arg name="rotation_speed" default="10" />
  <node name="rubot_nav" pkg="gopigo3_control" type="rubot_self_nav.py" output="screen" >
        <param name="distance_laser" value="$(arg distance_laser)"/>
        <param name="speed_factor" value="$(arg speed_factor)"/>
        <param name="forward_speed" value="$(arg forward_speed)"/>
        <param name="backward_speed" value="$(arg backward_speed)"/>
        <param name="rotation_speed" value="$(arg rotation_speed)"/>
  </node>
</launch>